src/main.py

```diff
@@ -284,14 +284,25 @@ def _run_testing(env: BaseEnv, approach: BaseApproach) -> Metrics:
         else:
             monitor = None
         try:
-            if CFG.env == "behavior" and \
-                CFG.behavior_option_model_eval:  # pragma: no cover
+            # Will evaluate all approaches on our simulator with the options
```

Comment:
given that EVERY settings configuration will get the code to this point, i feel like this comment should be more informative. i'll take a stab at rewriting it:

Now, measure success by running the policy in the environment. There are two special cases that we handle first. In the `if`, we consider the case where `plan_only_eval` is True, in which case we only check whether this BilevelPlanningApproach found a plan. In the `elif`, we consider the case where `behavior_option_model_eval` is True, in which case for BEHAVIOR we evaluate on option models instead of the low-level simulator. Finally, the `else` handles the default case, where we use `utils.run_policy` to roll out the policy in the environment.